src/main/java/com/firebolt/jdbc/connection/settings/FireboltProperties.java

```diff
@@ -236,6 +236,12 @@ public void addProperty(@NonNull String key, String value) {
 		switch (key) {
 			case "database": database = value; break;
 			case "engine": engine = value; break;
+			case "account_id":
+				if (accountId != null && !accountId.equalsIgnoreCase(value)) {
+					throw new IllegalStateException(format("Attempt to change account from %s to %s", accountId, value));
```

Comment:
The message is changed although I really do not understand why suggested message is better than mine. Also I removed "USE" from the message because this method is called when processing response from any command; passing the executed command here requires a lot of changes that I do not want to do.

Reply:
Firebolt has a preferred error message format, which is explained in detail in [here](https://app.getguru.com/card/i6jo6ngT/Firebolt-Error-Message-Style-Guide). The messaged from the design doc is following this guideline.
Please remember to comment on the design doc if you have questions or suggestions, this helps us adjust before all the implementations are completed.

Reply:
Again: too  many changes to match randomly written document.

Reply:
I added my comment to the document. Obviously this comment is write only.

Reply:
This document was written before we started the implementation and we follow it to keep the implementation logic consistent across the connectors. You can avoid doing "random" changes by reading the design document beforehand.